packages/jni-swig-stub/realm.i

```diff
@@ -346,6 +346,9 @@ import static io.realm.kotlin.internal.interop.realm_errno_e.*;
 // Array wrappers to allow building (continuous allocated) arrays of the corresponding types from
 // JVM
 %include "carrays.i"
+// Workaround for updated Swig behavior with 4.2.0
```

Comment:
Is there any reason why the workaround is split around the array_functions, instead of defining it just after `%include "carrays.i"`?

Reply:
It is just following the pattern from the [Swig changelog](https://github.com/swig/swig/commit/ecaa052f3d319834a66aaa07047be3662e5e52e2#diff-cd2fcc891412baae0fc46479c0870cbdd18133d06d68dcd216be8a37ecf77b37R10)

Reply:
Ahh I oversaw the line `... %array_functions and %array_class ...` all good!